src/checkpoint1/checkpoint1.html

```diff
@@ -0,0 +1,78 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>checkpoint 1</title>
+    <link rel="stylesheet" href="./reset.css">
+    <link rel="stylesheet" href="./checkpoint1.css">
+    <script src="https://kit.fontawesome.com/ceff186d6a.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <i class="far fa-envelope fa-lg"></i>
+        <i class="fas fa-list fa-lg"></i>
+        <i class="far fa-bell fa-lg"></i>
+        <i class="far fa-user-circle fa-lg"></i>
+    </header>
+
+    <div class="title">HYBRID</div>
+    
+    <aside>
+        <i class="far fa-envelope fa-lg"> Dashboard</i>
+        <i class="fas fa-list fa-lg"> UI Elements</i>
+        <i class="far fa-bell fa-lg"> Charts</i>
+        <i class="far fa-user-circle fa-lg"> Tabs & Panels</i>
+        <i class="far fa-envelope fa-lg"> Tables</i>
+        <i class="fas fa-list fa-lg"> Forms</i>
+        <i class="far fa-bell fa-lg"> List</i>
+        <i class="far fa-user-circle fa-lg"> Account</i>
+    </aside>
+    
+    <main>
+        
+    </main>
+
+    <div class="dash">Dashboard  <h6>Summary of your app</h6></div>
+
+    <div class="title"></div>
+
+    <nav>
+        <h4>Home /</h4>
+        <h4>Library /</h4>
+        <h4>Data</h4>
+    </nav>
+        <div class="one">
+            <i class="far fa-eye fa-lg"></i>
+            <h3>8,457</h3>
+            <h5>Daily Visits</h5>
+        </div>
+        <div class="two">
+            <i class="fas fa-shopping-cart fa-lg"></i>
+            <h3>52,160</h3>
+            <h5>Sales</h5>
+        </div>
+        <div class="three">
+            <i class="fas fa-comments fa-lg"></i>
+            <h3>15,823</h3>
+            <h5>Comments</h5>
+        </div>
+        <div class="four">
+            <i class="fas fa-user-friends fa-lg"></i>
+            <h3>36,752</h3>
+            <h5>No. of Visits</h5>
+        </div>   
```

Comment:
div.one,two,three,four all have the same style. so give them the same class so you only have to write it once in the css. you can keep a unique class for each element to use with their differences.